whois-update/src/main/resources/templates/notification.vm

```diff
@@ -78,6 +78,11 @@ $!update.reason
 Changed by SSO account: $update.preparedUpdate.update.effectiveCredential
 #end
 
+#if ($update.preparedUpdate.update.effectiveCredential && $update.preparedUpdate.update.effectiveCredentialType ==  'APIKEY')
+  Changed by API Key: $update.preparedUpdate.update.effectiveCredential
```

Comment:
Are you logging the API (access) key or the SSO account ? Maybe both if available, e.g.
```
Changed by SSO account: test@ripe.net with API key: XXXYYYZZZZ
```

Reply:
This is not for logging, It is for sending notification email to user incase it was successfully updated. I thiught you wanted only accessKey to be visible to outgoing emails. If not I can changed.

Reply:
You're correct, I meant the notification email, yes please change I think it'll be clearer, if you have both available.

Reply:
It takes only one variable adding two variables will require sme work as it is a common code , in that case it shoudl eb a seperate task as it is more of a nice to have feature

Reply:
Do you have the SSO account and the access key when calling the template?